src/httpserver.rs

```diff
@@ -179,6 +179,13 @@ async fn parse(req: Request<Body>) -> Result<(Bytes, String, String, String), ()
     }
 }
 
+/// Is `t` a valid GitHub event type? If this function returns `true` then it is guaranteed that `t`
+/// is safe to use in pathnames.
```

Comment:
`pathnames` -> `filesystem paths`?

So the event name is used later to build a path?

Reply:
Amusingly, I originally had "filesystem" in there, before realising that I was really just saying the same thing twice, and taking it out. I think it's pretty clear in this case that we could only possibly be talking about filesystems.

Reply:
Well, for a while I was confused as to whether you meant a part of the URL, before about 90% convincing myself you meant on the filesystem.

So if not `filesystem` then at least `path names` (two words).

Reply:
I take your point. Fixed in https://github.com/softdevteam/snare/pull/41/commits/8ced8f0b30c3f4dcc5e51e00c6bb0613b89a1c8d.